content/rule_book/quiz_process.md

```diff
@@ -1,106 +1,173 @@
+
+If only 2 teams were participating in a question and 1 team errs, the next question will be a bonus question, not a toss-up question. If only 1 team was participating in a question, it must have been a bonus question. As such, any outcome of the bonus question results in all 3 teams participating in the next question.
+
+### Bonus Questions
+
+If only 2 teams are participating in a question and 1 team errs, the next question will be an assigned bonus question for the team that did not err. An assigned bonus question is awarded to the corresponding chair of the chair that erred on the toss-up.
+
+#### Example
+
+There are 4 chairs per team, any 1 of which may be filled or empty. If the quizzer seated on chair 2 erred on the toss-up, the quizzer seated on chair 2 of the team that did not err on the toss-up will be awarded the bonus question. If no quizzer of the team that did not err on the toss-up is seated in chair 2, then the bonus is "for an empty seat" and considered a no jump.
+
+### Question Reading Structure
+
+A Quizmaster must introduce and read each question type in a specific, multipart way:
+
+1. Question Introduction
+    - State the full question number and type
+
+        > "Question number [question number] is a [type] question."
+
+    - Include whether it's a toss-up or bonus question
+
+        > "It is a ["toss-up" or "bonus"] for [team(s)]."
+
+    - For Situation questions only, include the question's interrogatives
+
+        > "I need to know [Situation question interrogatives]."
+
+2. Call the Question
+
+    > "Question number [question number], question:"
+
+3. Question text or remainder of question text
+
+A quizmaster may repeat or repeat as a paraphrase any component of the question introduction for additional clarity.
+
 ## Types of Quizzes
 
+There are 3 primary types of quizzes and 2 related overtime components of quizzes.
+
+### Logic
+
     Set quiz types to
         "3-Team 20-Question", "2-Team 15-Question Tie-Breaker",
         "2-Team 20-Question", "2-Team Overtime", and "3-Team Overtime".
 
-## Quizzes
+### 3-Team 20-Question Quiz
 
-There will be no duplicate questions in a single quiz.
+There will be 20 numeric questions in a quiz, excluding any overtime questions.
 
-**Participating in a Question**
-: A team that is eligible to jump in the question. A team that has no quizzers eligible to jump because of either foul out, err out, or quiz out, is still considered a team that is eligible to jump.
+A toss-up question is the result of an error by 1 team. The 2 remaining teams will have the opportunity to jump, but the team making the error is ineligible for that question. If question 1 to 15 is erred, the next question number will be a toss-up question. If question 16-20 is erred, and if 3 teams were participating on 16, the toss-up question will be question 16A.
```

Comment:
"the toss-up question will be appended with A"

Reply:
6a572c0